appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/util/ConnectionPoolReconfigHelper.java

```diff
@@ -147,7 +147,7 @@ private static ReconfigAction isEqualConnectorConnectionPool(ConnectorConnection
                 && !(newCcp.getDataStructureParameters().equals(oldCcp.getDataStructureParameters()))) {
             return ReconfigAction.RECREATE_POOL;
         }
-
+        
```

Comment:
It is a NetBeans thing - I think when you delete a space and then add it back in it auto-indents it. So I haven't actually deleted the whitespace.